frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -2,14 +2,16 @@ import React from "react";
 import { Card, Button, Row, Col } from "react-bootstrap";
 import cowHead from "./../../../assets/CowHead.png"; 
 
-// add parameters 
-const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
+
+// add parameters
+const ManageCows = ({userCommons, commons, onBuy, onSell}) =>  {
     // update cowPrice from fixture
+    // Stryker disable ArithmeticOperator : line 24: not very important and unsure of how to test this
```

Comment:
Why is this here?   This keeps showing up in this team's PRs, and I'm skeptical.  I need to be convinced that this is necessary, especially since "line 24" seems to be misleading.